packages/node/src/utils/substrate.ts

```diff
@@ -423,17 +434,25 @@ export async function fetchLightBlock(
     block: { header },
     events: events.toArray(),
   };
-
   return {
-    block: blockHeader,
-    events: events.map((evt, idx) => merge(evt, { idx, block: blockHeader })),
+    block: {
+      block: blockHeader,
+      events: events.map((evt, idx) => merge(evt, { idx, block: blockHeader })),
+    },
+    getHeader: () => {
+      return {
+        hash: blockHeader.block.header.hash.toString(),
+        height: blockHeader.block.header.number.toNumber(),
+        parentHash: blockHeader.block.header.parentHash.toString(),
+      };
```

Comment:
This can be put into a function to dedupe with https://github.com/subquery/subql/pull/2257/files#diff-622aec34f349237d6d2b7b613e44aedbe25468c88ba6597f2051bb9c825d55f4R396-R398

Reply:
Might be a good idea to unify getHeader: () return type with `Header` https://github.com/subquery/subql/pull/2257/files#diff-8138bbd5bd4b60a1dea77c9b7b6577d99ff57d83838c9a49fe66024a1793b153R23, so `substrateHeaderToHeader ` can be re-used